projects/demo/src/pages/recipes/prefix/examples/1-pattern-mask/component.ts

```diff
@@ -0,0 +1,24 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+
+import mask from './mask';
+
+@Component({
+    selector: 'prefix-doc-example-1',
+    template: `
+        <tui-input
+            ngModel=""
```

Comment:
Maybe we should add proper ngModel binding for the example?

Reply:
fixed